README.md

```diff
@@ -630,6 +630,7 @@ Logbook comes with a convenient auto configuration for Spring Boot users. It set
 - Header-/Parameter-/Body-Filters
 - HTTP-/JSON-style formatter
 - Logging writer
+- A `RestTemplate`-friendly `ClientHttpRequestInterceptor`

```

Comment:
I'd say the new module deserves a separate section in the README, similar to the other integrations.